modules/ROOT/pages/managing-users.adoc

```diff
@@ -90,9 +90,20 @@ If enabled, the SAML assertions sent from the IdP must be encrypted and follow t
 +
 . Sign out of Anypoint Platform, navigate to the sign-on URL you entered in the *Identity Management SAML 2.0* form, then sign in through your identity provider to test the configuration.
 
+== Import SAML Metadata
+
+You can import your identity provider's SAML 2.0 metadata to Anypoint Platform. You can use *Import IdP Metadata* to import an XML file that contains a SAML 2.0 configuration from your identity provider. The XML file's values populate the *Identity Management SAML 2.0* form.
+
+When you upload an XML file to import your metadata:
+
+* The file must be smaller than 16 KB.
+* The file must contain values that correspond to the fields in the *Identity Provider* page.
+* When you upload an XML file that contains more than one identity configuration, Anypoint Platform uses the first available identity provider configuration.
```

Comment:
"If the file contains more than one entry, Anypoint Platform uses the first identity provider configuration."

Reply:
```suggestion
* If you upload an XML file that contains more than entry, Anypoint Platform uses the first identity provider configuration.
```